iframe/index.html

```diff
@@ -51,13 +51,13 @@ <h2 class="form-title font-bold text-2xl text-center mb-6">Authsio - Login</h2>
 
         <fieldset class="grid grid-template-columns-1 px-5 mb-4">
           <label for="password" class="block mb-2 ml-1 label text-sm font-bold uppercase">Password</label>
-          <input type=" password" class="form-input peer rounded border-0 w-full px-4 py-2 px-3  
+          <input type="password" class="form-input peer rounded border-0 w-full px-4 py-2 px-3  
```

Comment:
This change is needed but nothing around this PR is related. If you can submit this change with only this one line change we can get it in place.